packages/node_modules/node-red/red.js

```diff
@@ -133,9 +133,14 @@ try {
 if (parsedArgs.verbose) {
     settings.verbose = true;
 }
-if (parsedArgs.safe) {
+if (parsedArgs.safe || process.env.NODE_RED_ENABLE_SAFE_MODE) {
```

Comment:
We might also want to check for string "false" 
so if someone does set it to "false" it does the right thing...